### ISSUE TITLE
Error Broadcasting signed Tx's

### ISSUE THREAD

**Issue by robertandrewsmith** [MyCryptoHQ/MyCrypto#2481]
I created and signed a tx offline using Windows version 1.6.3

I attempted to broadcast a tx created with MyCrypto, on etherscan.io (since the "send" button was missing from the Broadcast Tx page of MyCrypto Windows version 1.6.3.

Etherscan returned the following error below.  I will also reach out to Etherscan, but it could also be a problem with MyCrypto.

Error! Unable To Broadcast TX : {"jsonrpc":"2.0","id":1,"error":{"code":-32602,"message":"invalid argument 0: json: cannot unmarshal invalid hex string into Go value of type hexutil.Bytes"}}

---

### Description of Issue

...

### What exactly were you doing when you encountered this issue?

...Give us some steps to reproduce...

### How were you accessing your wallet?

...Ledger, Trezor, MetaMask, Private Key, Keystore File, Parity Signer, Mnemonic Phrase?...

### Description of Your Machine

- Browser:
- Browser version:
- Operating system:
- Device:
- MyCrypto.com vs Desktop Application?
- MyCrypto Version Number:

### Console Logs / Screenshots

...

**Comment by tayvano** [comment]
We're you using MyCrypto to generate and sign the transaction?

Can you share what you pasted in Etherscan and the Broadcast TX page that produced that error?

**Comment by robertandrewsmith** [comment]
SORRY Tay - I made a copy/paste error.  BUT I am having the same problem as I did before the most recent update: (generating and signing offline, then broadcasting online using MyCrypto.)
The Tx shows as "broadcast", but not pending in etherscan or etherchain. Though it says pending in MyCrypto.  Sent with 8 gwei and 21K gas.
Here's the signed Tx: 0x81b0feca9e709c7b10c077e533c3946d21337a575414eb03897e02929b274796
